server/src/controllers/food.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from "express";
+import FoodService from "../services/food";
+
+export async function create(request: Request, response: Response) {
+  try {
+    const food = request.body.food;
+    const createdFood = await FoodService.create(food);
+    response.status(201);
+    response.json({ food: createdFood });
+  } catch (error) {
+    response.status(400);
+    response.json(error);
+  }
+}
+
+export async function createMany(request: Request, response: Response) {
+  try {
+    const foods = request.body.foods;
+    const createdFoods = await FoodService.createMany(foods);
+    response.status(201);
+    response.json({ foods: createdFoods });
+  } catch (error) {
+    response.status(400);
+    response.json(error);
+  }
+}
+
+export async function deleteFoodById(request: Request, response: Response) {
+  try {
+    const foodID = request.body.foodID;
```

Comment:
**Wont be needing this**
A delete request won't need a body if its just one piece of information
we can leave the id in the request params like so:
`delete('foods/:id')`